### ISSUE TITLE
Rename remove

### ISSUE THREAD

**Issue by prat0088** [TailorDev/Watson#344]
Remove implies taking an item out of a collection, but not destroying it.  Delete implies both.  The behavior of watson remove actually delete, but it's named remove.  

I think it would be clearer if the remove command was renamed delete.

**Comment by k4nar** [comment]
That's true. Maybe we could alias them? Otherwise that would be a rather big breaking change.

**Comment by jmaupetit** [comment]
Yup, renaming `remove` to `delete` and aliasing it would be nice!
